### ISSUE TITLE
Literal is not instantiable

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1398]
```python
Literal[1](1)
```

This should generate an error. Literal cannot be instantiated at runtime.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.104, which I just published. It will also be included in the next release of pylance.
